packages/andromeda-non-fungible-tokens/src/crowdfund.rs

```diff
@@ -20,8 +20,10 @@ pub enum ExecuteMsg {
     Mint(Vec<CrowdfundMintMsg>),
     /// Starts the sale if one is not already ongoing.
     StartSale {
+        /// When the sale start. Defaults to current time.
+        start_time: Option<u64>,
         /// When the sale ends.
-        expiration: Expiration,
+        duration: u64,
```

Comment:
Would it make sense to have this be an end time rather than duration?

Reply:
@leifasorensen @SlayerAnsh 

What is a better UX here? Duration for crowdfund or a set expiration time?

Reply:
Should we rename `expiration` to `end_time`? Since we already have `start_time`

Reply:
> Should we rename `expiration` to `end_time`? Since we already have `start_time`

That makes sense to me